_data/testimonials.yml

```
@@ -12,3 +12,17 @@
   toolId: ttt
   story: |
     The ELIXIR-GOBLET train the trainer course is an excellent opportunity to up-skill myself to designing and developing better course syllabi and material as well as smoothen the delivery of my courses. I had the incredible opportunity to participate in the ELIXIR-GOBLET Train the Trainer course, and it has been a transformative experience for my career as an educator. The course seamlessly blends theory with practical application, providing a comprehensive understanding of effective training methodologies.
+- author:
+    name: Fotis E. Psomopoulos
+    affiliation: on behalf of EOSC4Cancer
+    imageUrl: https://i.pravatar.cc/150?u=emily-johnson-2
```

```suggestion
    imageUrl: https://github.com/fpsom.png
```
This will show Fotis github photo instead of the generic image